src/15_classes.py

```diff
@@ -2,28 +2,41 @@
 # constructor
 
 # YOUR CODE HERE
-        
+class LatLon:
+    def __init__(self, lat, lon):
+        self.lat = lat
+        self.lon = lon
 # Make a class Waypoint that can be passed parameters `name`, `lat`, and `lon` to the
 # constructor. It should inherit from LatLon. Look up the `super` method.
-
 # YOUR CODE HERE
-
+class Waypoint(LatLon):
+    def __init__(self, name, lat, lon):
+        super().__init__(lat, lon)
+        self.name = name
+    def __str__(self):
+        return f'{self.name} at {self.lat} and {self.lon}'
 # Make a class Geocache that can be passed parameters `name`, `difficulty`,
 # `size`, `lat`, and `lon` to the constructor. What should it inherit from?
 
 # YOUR CODE HERE
-
+class Geocache(Waypoint):
+    def __init__(self, name, difficulty, size, lat, lon):
+        super().__init__(name, lat, lon)
+        self.difficulty = difficulty
+        self.size = size
+    def __str__(self):
+        return f'{self.name} diffculty level: {self.difficulty}, size: {self.size} at {self.lat} and {self.lon}'
 # Make a new waypoint and print it out: "Catacombs", 41.70505, -121.51521
 
```

Comment:
Superb work with the basics of OOP in Python. How do you like this syntax compared with JS?